worlds/openrct2/test/__init__.py

```diff
@@ -0,0 +1,5 @@
+from test.bases import WorldTestBase
+
+
+class CV64TestBase(WorldTestBase):
```

Comment:
CastleVania64 Test Base? 
since this seems to be just empty you can remove this or if you need it, rename it to OpenRCT2TestBase but test bases are only needed if you need to do some commen setup across multiple tests, if not you can just use `WorldTestBase`